packages/ethereum/contracts/src/Channels.sol

```diff
@@ -140,17 +128,16 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer, Multicall {
   struct TicketData {
     bytes32 channelId;
     Balance amount;
-    TicketIndex index;
+    TicketIndex maxTicketIndex;
+    TicketIndexOffset indexOffset;
```

Comment:
What is the offset used for?

Reply:
changed, added some documentation and better naming